bindata/network/openshift-sdn/sdn-ovs.yaml

```diff
@@ -139,33 +139,32 @@ spec:
         livenessProbe:
           exec:
             command:
-            # Validate that ovs-vswitchd and ovsdb are running. These three commands check
-            # 1. that the pids in the pid file are up
-            # 2. that ovs-vswitchd is responding to queries
-            # 3. that ovsdb is responding to queries
-            # 4. if br0 is configured, that its management process is responding
+            # Validate that ovs-vswitchd and ovsdb are running. Check that
+            # 1. that ovs-vswitchd is responding to queries
+            # 2. that ovsdb is responding to queries
+            #
+            # Need the manual target file because ovs-appctl doesn't like
+            # being in a different namespace from ovs-vswitchd.
             - /bin/bash
             - -c
             - |
               #!/bin/bash
-              /usr/bin/ovs-appctl -T 5 ofproto/list > /dev/null &&
-              /usr/bin/ovs-vsctl -t 5 show > /dev/null &&
-              if /usr/bin/ovs-vsctl -t 5 br-exists br0; then /usr/bin/ovs-ofctl -t 5 -O OpenFlow13 probe br0; else true; fi
+              PID=$(cat /var/run/openvswitch/ovs-vswitchd.pid)
+              /usr/bin/ovs-appctl -t "/var/run/openvswitch/ovs-vswitchd.${PID}.ctl" -T 5 ofproto/list > /dev/null &&
+              /usr/bin/ovs-vsctl -t 5 show > /dev/null
```

Comment:
we don't care for the br0 to exist anymore? I thought that was needed for liveness to make sure new pods can come up and flows be configured for them?

Reply:
that doesn't affect openvswitch readiness though. It's not a useful check, really.